src/components/CommentCard.jsx

```diff
@@ -0,0 +1,24 @@
+import "../styles/comment-card.css";
+
+const CommentCard = ({ comment }) => {
```

Comment:
This is not ticket 6, please be careful not to do stuff for the later tickets :)